wordpress-mobile/WordPress-Android
# Moved vector drawables used in notifications quick actions

...to the specific folder `/res/drawable-anydpi-v21`

For reference read https://developer.android.com/guide/topics/resources/providing-resources.html#AlternativeResources

Also this PR bumps the gradle tools version being used

Fixes #5635 

<img width="588" alt="screen shot 2017-04-17 at 10 35 35 pm" src="https://cloud.githubusercontent.com/assets/6597771/25111114/6f837f30-23bf-11e7-8c67-70514ba35ec1.png">
<img width="588" alt="screen shot 2017-04-17 at 10 35 56 pm" src="https://cloud.githubusercontent.com/assets/6597771/25111141/8f819196-23bf-11e7-86c0-6af90ee26f0b.png">
<img width="250" alt="screen shot 2017-04-17 at 10 36 17 pm" src="https://cloud.githubusercontent.com/assets/6597771/25111149/99a122a4-23bf-11e7-9de7-b91eed9b2e6f.png">


To test:
1. Open the app on a pre-21 device - specifically we saw this problematic behavior on a Samsung Ta 3 (7.0), Android 4.2.2
2. make a notification be received, by commenting on the web a post of the user logged in in the tablet.
3. the notification should be received and shown without a problem


cc @maxme @hypest
